regexes/device/mobiles.yml

```diff
@@ -25479,11 +25506,11 @@ KREZ:
 
 # KRIP (https://www.krip.com/)
 KRIP:
-  regex: 'K4m(?:[);/ ]|$)'
+  regex: 'K(?:4m?|5[bdm]?|6b|55h|57|65)(?:[);/ ]|$)'
```

Comment:
I have doubts about K4, K5

Movic K4 -  Android 8,  device code `K4`
KRIP K4 - Android 8, device code `K4`

EKINOX  K5 - Android 8, device code `K5`
KRIP K5 - Android 8, device code `K5`
Iku Mobile K5 - Android 10, device code `K5`
Kubo K5 - Android ???, device code `K5`
Lesia K5 - Android ~8, device code `K5`

We could create such a rule `Android 8.+; K[45](?: Build|[);])`, but it won't help now (too many matches  OS versions)
I propose to exclude these 2 devices for now, and move the tests to unknown.yml file